<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LTU6D8L" failures="3" tests="4" name="BankingApp Test(failed)" time="42.145" errors="0" timestamp="2022-04-30T22:58:09 IST">
  <testcase classname="com.BankingMini.testCases.TC_LoginTest_001" name="loginTest" time="3.008"/>
  <testcase classname="com.BankingMini.testCases.TC_LoginDDT_002" name="loginDDT" time="6.44">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.BankingMini.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.BankingMini.testCases.TC_LoginDDT_002" name="loginDDT" time="7.103">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.BankingMini.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.BankingMini.testCases.TC_LoginDDT_002" name="loginDDT" time="6.706">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.BankingMini.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test(failed) -->
